stable/prometheus-operator/templates/exporters/kubelet/servicemonitor.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.kubelet.enabled }}
-apiVersion: monitoring.coreos.com/v1
+apiVersion: {{ printf "%s/v1" (.Values.prometheusOperator.crdApiGroup | default "monitoring.coreos.com") }}
```

Comment:
Support for changing the crdApiGroup was removed entirely in prometheus-operator, so supporting it is not necessary.